feat: pull logo from tmdb

markhc opened the pull request:
Credits to Uploadrr which already has this feature and made me want to implement it here.

What this PR does:
- Adds 2 config options to control whether to pull logos from TMDB, and the logo size.
- If enabled, it will pull the first logo for the TV/Movie from TMDB using the `/{tmdbId}/images` endpoint and place it at the top of the BBCode description.

By default the option is disabled and no extra requests are sent to TMDB.

What it looks like:

![image](https://github.com/user-attachments/assets/7809d42c-5aad-4599-a009-86252ffa9775)

Let me know if this is unwanted or what can be made to improve it if necessary. 

Cheers!

Audionut commented:
Nice.
Can you add an option for language matching (2 digit code ('en' default))?

The first movie I happened to check only had 'cs' logos which doesn't fit well in en only sites. Allowing to specify the 2 digit language codes though (instead of hardcoded en default) will give non-English sites some love.

It might even be worthwhile for a fallback to 'en' when other language preferred but not available. I'm happy to push without this last idea.

markhc commented:
Good idea. Added basic support for an additional language, with fallback to English. 

Can be easily expanded to support multiple languages (by making the config entry an array instead of a str) but I am not sure if there's any use case for that, so I kept it simple.

![image](https://github.com/user-attachments/assets/d59e882d-35f8-4bed-b946-77722f7cff43)

Also added a debug print and extracted the code into it's own function to reduce duplication.

Audionut commented:
If you're happy with my quick checks, I'm happy also.

markhc commented:
Yep. All good! 👍
